examples/inverse_folding/README.md

```diff
@@ -0,0 +1,172 @@
+# Inverse folding with ESM-IF1
+
+The ESM-IF1 inverse folding model is built for predicting protein sequences 
+from their backbone atom coordinates. We provide scripts here 1) to sample sequence 
+designs for a given structure and 2) to score sequences for a given structure. 
+
+Trained with 12M protein structures predicted by AlphaFold2, the ESM-IF1
+model consists of invariant geometric input processing layers followed by a
+sequence-to-sequence transformer, and achieves 51% native sequence recovery on
+structurally held-out backbones with 72% recovery for buried residues.
+The model is also trained with span masking to tolerate missing backbone
+coordinates and therefore can predict sequences for partially masked structures.
+
+More details in our bioRxiv [pre-print](https://doi.org/10.1101/2022.04.10.487779).
+
+![Illustration](illustration.png)
+
+## Recommended environment
+It is highly recommended to start a new conda environment from scratch due to
+potential CUDA compatability issues between pytorch and the pytorch-geometric
+package required for the inverse folding model.
+
+To set up a new conda environment with required packages,
+
+```
+conda create -n inverse python=3.9
+conda activate inverse
+conda install pytorch cudatoolkit=11.3 -c pytorch
+conda install pyg -c pyg -c conda-forge
+conda install pip
+pip install biotite
+pip install fair-esm
```

Comment:
Here I'd mirror the notebook instructions instead, as we'll want to incorporate any bugfixes before tagging this as a new release:

> Work with the current repository state (includes any updates since last release):
> ```bash
> $ git clone https://github.com/facebookresearch/esm.git
> $ cd esm; pip install .
> ```

Reply:
Changed to: pip install git+https://github.com/facebookresearch/esm.git